.travis.yml

```diff
@@ -21,6 +25,14 @@ env:
   global:
     - CFLAGS="-Werror -Wall -Wextra -Wno-error=sign-compare -Wno-error=unused-parameter -Wno-error=missing-field-initializers"
 
+before_install:
+  - |
+    if [ "$TRAVIS_OS_NAME" == freebsd ]; then
+      sudo pkg install -y evdev-proto libgudev libxml++
+      sudo pkg install -y automake libtool pkgconf gmake
```

Comment:
TravisCI have these dependencies preinstalled, so I can drop it to avoid confusion. For example, `bash` used to run libwacom test scripts is actually what TravisCI use to run `before_install` target.